include/cgimap/util.hpp

```diff
@@ -90,6 +90,43 @@ inline bool iequals(std::string_view a, std::string_view b) {
   return a.size() == b.size() &&
          std::equal(a.begin(), a.end(), b.begin(), b.end(), ichar_equals);
 }
+
+// Remove leading and trailing whitespace from string
+inline std::string_view trim(std::string_view str) {
+  auto start = str.find_first_not_of(" \t\n\r");
+  if (start == std::string::npos)
+    return {};
+  auto end = str.find_last_not_of(" \t\n\r");
+  return str.substr(start, end - start + 1);
+}
+
+inline std::vector<std::string_view> split_trim(std::string_view str, char delim) {
```

Comment:
This could be replaced later on by a c++20 version: https://github.com/mmd-osm/openstreetmap-cgimap/blob/cpp20_1/include/cgimap/util.hpp#L104-L117